### PR TITLE
docs: Update document of cosmovisor

### PR DESCRIPTION
**Pull request by gyuguen**
Cosmovisor documentation needs updating, so I have made the following changes.

### CODE REVIEW

**Review by H4NLee — APPROVED** [review]
Thank you.

And I think this guide should be for general usage, not a specific version of panacea.
I think there is no need of example in `Cosmovisor Setup`. `Example` seems to be enough.
wdyt?

**Comment by gyuguen** [comment]
> Thank you.
> 
> And I think this guide should be for general usage, not a specific version of panacea. I think there is no need of example in `Cosmovisor Setup`. `Example` seems to be enough. wdyt?

You're right. I'll think about it a little bit more.

**Comment by gyuguen** [comment]
I've made changes based on your comments.
https://github.com/medibloc/panacea-core/pull/644/commits/7fafc1f14d77d13c6f6382b815819efba73e847d
Please tell me your feedback :)

File: .gitbook/4-validators/4-cosmovisor.md [hunk 1]
```diff
@@ -31,29 +33,49 @@ There are four variables. (two are requirement, and two are optional)
...
+
 ```shell
 export DAEMON_HOME=$HOME/.panacea
 export DAEMON_NAME=panacead
 export DAEMON_RESTART_AFTER_UPGRADE=true
 export DAEMON_ALLOW_DOWNLOAD_BINARIES=false
 ```
 
-Create a directory for the genesis binary and upgrade binary.
+Create a directory for the genesis binary. This path should contain the panacead currently running on the mainnet.
```
Comment (H4NLee) [inline .gitbook/4-validators/4-cosmovisor.md:69]:
```suggestion
Create a directory for the genesis binary. The current version of binary should be under the directory.
```

File: .gitbook/4-validators/4-cosmovisor.md [hunk 2]
```diff
@@ -64,103 +86,30 @@ make clean && make build
...
-cd panacea-core
+## Upgrade chain
 
-make clean && make build
-cp ./build/panacead $DAEMON_HOME/cosmovisor/genesis/bin
-```
+cosmovisor supports chain upgrades automatically. This requires some preliminary work.
 
-Create a new key for the validator, then add a genesis account and transaction:
+You need to create a path in cosmovisor's home path to store the binary to be upgraded.
```
Comment (H4NLee) [inline .gitbook/4-validators/4-cosmovisor.md:100]: suggest to use the same word with above
```suggestion
You need to create a directory for the binary to be upgraded.
```

File: .gitbook/4-validators/4-cosmovisor.md [hunk 3]
```diff
@@ -64,103 +86,30 @@ make clean && make build
...
---title upgrade \
---description upgrade \
---upgrade-height 30 \
---from validator \
---chain-id test \
---yes
-
-./build/panacead tx gov deposit 1 10000000umed --from validator --chain-id test --yes
-./build/panacead tx gov vote 1 yes --from validator --chain-id test --yes
+Generate a new binary and navigate to the path you created above.
```
Comment (H4NLee) [inline .gitbook/4-validators/4-cosmovisor.md:107]:
```suggestion
Generate a new binary and navigate to the directory you created above.
```

File: .gitbook/4-validators/4-cosmovisor.md [hunk 4]
```diff
@@ -64,103 +86,30 @@ make clean && make build
...
 ```
 
-The upgrade will be performed automatically at height 30.
-
-```shell
-cosmovisor version
-2.0.2 # output
-```
\ No newline at end of file
+The upgrade is complete when the chain is stopped by SoftwareUpgradeProposal, the binaries are replaced by cosmovisor, and the chain is started.s
```
Comment (H4NLee) [inline .gitbook/4-validators/4-cosmovisor.md:115]:
```suggestion
The upgrade is complete when the chain reaches the target height of upgrade. The binary is replaced by cosmovisor automatically, and the chain is started.
```

**Review by H4NLee — APPROVED** [review]
I left minor comments. LGTM!

**Review by inchori — APPROVED** [review]
LGTM 👍

**Review by audtlr24 — APPROVED** [review]
lgtm

**Review by youngjoon-lee — APPROVED** [review]
lgtm

**Pull request merged**